### PR TITLE
Initial Azure.ResourceManager.Relay

### LINKED ISSUE #29285: Initial Azure.ResourceManager.Relay
https://github.com/Azure/azure-rest-api-specs/tree/main/specification/relay/resource-manager

### PR DESCRIPTION
**Pull request by Yao725**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/relay/test-resources.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
```
Comment (ArthurMa1978) [inline sdk/relay/test-resources.json:2]: remove this file as we don't need to preview test env

File: sdk/relay/Azure.ResourceManager.Relay/src/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Runtime.CompilerServices;
+
+[assembly: Azure.Core.AzureResourceProviderNamespace("Relay")]
```
Comment (ArthurMa1978) [inline sdk/relay/Azure.ResourceManager.Relay/src/Properties/AssemblyInfo.cs:6]: Microsoft.Relay

File: sdk/relay/Azure.ResourceManager.Relay/src/autorest.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+csharp: true
+library-name: Relay
+namespace: Azure.ResourceManager.Relay
+require: https://github.com/Azure/azure-rest-api-specs/tree/cc8796418bed73e7e3755d8a6a2d84abcb3ec7f4/specification/relay/resource-manager/readme.md
+output-folder: $(this-folder)/Generated
+clear-output-folder: true
+skip-csproj: true
+ 
+
+rename-rules:
```
Comment (ArthurMa1978) [inline sdk/relay/Azure.ResourceManager.Relay/src/autorest.md:17]: update to the latest template:
output-folder: $(this-folder)/Generated
clear-output-folder: true
skip-csproj: true
modelerfour:
  flatten-payloads: false

format-by-name-rules:
  'tenantId': 'uuid'
  'etag': 'etag'
  'location': 'azure-location'
  '*Uri': 'Uri'
  '*Uris': 'Uri'

**Review by ArthurMa1978 — APPROVED** [review]
Please fix comments

File: sdk/relay/Azure.ResourceManager.Relay/src/Properties/AssemblyInfo.cs (hunk 2 above)
Comment (Yao725) [reply sdk/relay/Azure.ResourceManager.Relay/src/Properties/AssemblyInfo.cs]: Fixed

File: sdk/relay/test-resources.json (hunk 1 above)
Comment (Yao725) [reply sdk/relay/test-resources.json]: Removed

**Pull request merged**